src/cli/main.pl

```diff
@@ -1077,6 +1077,12 @@
            shortflags([i]),
            default('_'),
            help('id of document to retrieve')],
+          [opt(ids),
+           % todo something something comma separated values
```

Comment:
? :O

Reply:
the cli now requires you to pass in ids like so:
./terminusdb doc get admin/foo --ids '["Foo/1","Foo/2"]'

So as a json list.

Would probably be nicer if you could directly specify a comma separated list.